Fauna-RPG/Entity/Components/StatComponent.gd

```diff
@@ -6,8 +6,4 @@ class_name StatComponent
 ## have different stats based on the Entity type.
 
 
-var stats: EntityStats
-
-
-func _ready() -> void:
-	stats = EntityStats.new()
+var stats: EntityStats = EntityStats.new()
```

Comment:
I prefer having a `_ready` function so that we don't have to remember that this specific variable doesn't have one and is directly initialized

Reply:
This will change with the stat changes mentioned above, so can wait until then to fix